packages/connect-form/src/components/advanced-options-tabs/authentication-tab/authentication-plain.tsx

```diff
@@ -1,9 +1,70 @@
 import React from 'react';
+import { TextInput } from '@mongodb-js/compass-components';
+
+import ConnectionStringUrl from 'mongodb-connection-string-url';
+import { UpdateConnectionFormField } from '../../../hooks/use-connect-form';
+import FormFieldContainer from '../../form-field-container';
+import {
+  ConnectionFormError,
+  errorMessageByFieldName,
+} from '../../../utils/validation';
+import {
+  getConnectionStringPassword,
+  getConnectionStringUsername,
+} from '../../../utils/connection-string-helpers';
+
+export const PLAIN_USERNAME_LABEL = 'Username';
+export const PLAIN_PASSWORD_LABEL = 'Password';
+
+function AuthenticationPlain({
+  connectionStringUrl,
+  updateConnectionFormField,
+  errors,
+}: {
+  connectionStringUrl: ConnectionStringUrl;
+  errors: ConnectionFormError[];
+  updateConnectionFormField: UpdateConnectionFormField;
+}): React.ReactElement {
+  const username = getConnectionStringUsername(connectionStringUrl);
+  const password = getConnectionStringPassword(connectionStringUrl);
+  const usernameError = errorMessageByFieldName(errors, 'ldapUsername');
+  const passwordError = errorMessageByFieldName(errors, 'ldapPassword');
 
-function AuthenticationPlain(): React.ReactElement {
   return (
     <>
-      <p>LDAP</p>
+      <FormFieldContainer>
+        <TextInput
+          onChange={({
+            target: { value },
+          }: React.ChangeEvent<HTMLInputElement>) => {
+            updateConnectionFormField({
+              type: 'update-username',
+              username: value,
+            });
+          }}
+          label={PLAIN_USERNAME_LABEL}
```

Comment:
Similar question here to the last one, do we want to add an info link like we currently have in Compass?
<img width="467" alt="Screen Shot 2022-01-26 at 12 53 16 PM" src="https://user-images.githubusercontent.com/1791149/151219007-cfccd106-3327-4a90-98df-fd65689b608f.png">
links to https://docs.mongodb.com/manual/core/security-ldap/

Reply:
I was chatting with Claudia about these, considering that it would require a "custom component" I would remove all of them. Possibly replacing it with a more generic help for the whole auth method.

Although I would argue that our goal should not be to teach our users what LDAP is and how it works but only make them know how to connect to MongoDB with LDAP.

If you look into that page, i do not believe that a user would get any real help from that, that's all the "client" part says about it: https://docs.mongodb.com/manual/core/security-ldap/#connect-to-a-mongodb-server-via-ldap-authentication

Reply:
We use the info icon on a text input for the authentication database already:
<img width="367" alt="Screen Shot 2022-01-26 at 1 11 07 PM" src="https://user-images.githubusercontent.com/1791149/151221796-d675ca4b-c397-420e-9954-8537ba7a1323.png">
I see what you mean, the current link probably doesn't help people out much. I'm cool with removing it 👍